pidtree_bcc/plugins/loginuidmap.py

```diff
@@ -0,0 +1,35 @@
+import pwd
+import sys
+
+from pidtree_bcc.plugin import BasePlugin
+
+
+class Loginuidmap(BasePlugin):
+    """ Plugin for mapping PID to loginuid and username """
+
+    NO_LOGINUID = 4294967295  # unsigned -1
+
+    def process(self, event):
```

Comment:
Seeing as you've already made everything else nice and typed, could you do the same here?